shared/color.js

```diff
@@ -0,0 +1,4 @@
+export const whiteTheme = "#EAEFF2";
```

Comment:
1. Color is not a theme, but part of a theme and there's usually 1 theme for application (or 2-3 if app supports dark mode and accessible mode).
2. Let's put colors in an object, like:
```
colors = {
  white: "#EAEFF2",
  green: ......
}
```
3. It would be cool if we had some ThemeProvider wrapping our entire app so we don't need to explicitly import it, but have it available in the props (something like ThemeProvider from Styled Components - https://www.styled-components.com/docs/advanced#theming)
4. These colors are ugly :P

Reply:
i think wrappers should be as new task